gips/data/merra/install_basic_client.py

```diff
@@ -0,0 +1,60 @@
+import cookielib
+import netrc
+import urllib2
+import re
+ 
+import pydap.lib
+from pydap.exceptions import ClientError
+ 
+ 
+# Set the debug level for urllib2.
+debuglevel=1
+
+def install_basic_client(uri='', user='', passwd='', use_netrc=True):
```

Comment:
This function could use a docstring.  All functions should get one, first of all, and second, you seem to be altering a library's internals with a monkeypatch.  The stranger the code, the more important the docstring I think.  Also I'd add a link to where you got the technique, which seems to be in pydap's docs?   Maybe here? --> 
http://pydap.readthedocs.io/en/latest/client.html?highlight=request#configuring-a-proxy